DevelopedCalculator.js

```diff
@@ -25,6 +24,7 @@ class DevelopedCalculator extends Calculator {
     }
     gettingHistoryArray() {
         this.history.join(",");
+        this.historyElement.textContent = this.currentResult;
```

Comment:
This part `this.historyElement.textContent =` is correct 
But you are assigning the wrong value.
Do we need the current result or the history content?

Reply:
Maybe I pushed the updates in the same time of pushing your comment